openapi/external_api.yaml

```diff
@@ -333,11 +362,24 @@ components:
           type: string
           format: uri
 
-    HTTPReconciliationOperations:
+    HTTPReconciliationInfo:
       type: object
+      required: [ lock, runtimeID, schedulingID, created, updated, status,operations ]
       properties:
-        cluster:
-          $ref: "#/components/schemas/cluster"
+        lock:
```

Comment:
please remove this and add the `finished` field

Reply:
Replaced